proto/api/v1/workflow.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "google/rpc/status.proto";
+
+message ExecuteWorkflowRequest {
+  // URL of the repo the workflow is running for
+  // Ex. "https://github.com/some-user/acme"
+  string repo_url = 1;
+  // Branch the workflow should run on
+  // Ex. "cool-feature"
+  string branch = 2;
+
+  // OPTIONAL FIELDS
+
+  // Name of the workflow action to execute. Corresponds to an action in
+  // buildbuddy.yaml, or an action name in the default workflow config if
+  // buildbuddy.yaml is missing from the repo.
+  //
+  // If this is not set, the API will execute all actions in the workflow
+  //
+  // Ex. "Test all targets"
+  string action_name = 3;
```

Comment:
I noticed that the response could contain multiple actions, should we make this a list of action instead of a single action?

Reply:
Added in this PR: https://github.com/buildbuddy-io/buildbuddy/pull/3805